### PR TITLE
Update TabletPageScreen.java

### PR DESCRIPTION
**Pull request by ErythroCraft**
editing nested code block

### CODE REVIEW

**Comment by Mrbysco** [comment]
Is there any reason for these changes? Feels nit-picky

**Comment by ErythroCraft** [comment]
> Is there any reason for these changes? Feels nit-picky

sure it is, but it's going faster

So I found many parts I'll editing

**Comment by ErythroCraft** [comment]
@Mrbysco END

- (java:S1126) 
- sonarlint(java:S6541) + sonarlint(java:S3776)

will not edit today

**Comment by TelepathicGrunt** [comment]
@ErythroCraft None of your changes actually will improve performance. The code is basically going to be optimized to the same or nearly same bytecode by JIT. All I really see here is just code style changes. That's it. No change that actually reduces workload in a measurable way to the player.

However, one thing I do see is several spots changes the existing calculations due to the new parenthesis and thus, will change the behavior of this screen. Have you tested this in-game to verify that the new order of operations doesn't actually break the screen in unintended ways from past behavior?

File: src/main/java/elucent/rootsclassic/client/screen/TabletPageScreen.java [hunk 1]
```diff
@@ -137,42 +135,43 @@ public void render(GuiGraphics guiGraphics, int mouseX, int mouseY, float partia
...
-        //        RenderSystem.setShaderTexture(0, Const.tabletGui);
+      case TYPE_NULL -> {
+				//	text only
+        //	Roots.logger.info("null type ")
+        //	RenderSystem.setShaderTexture(0, Const.tabletGui)
         guiGraphics.blit(Const.TABLETGUI, basePosX, basePosY, 64, 0, 192, 256);
         info = page.makeLines(makeInfo());
         for (int i = 0; i < info.size(); i++) {
-          textLines.add(new ScreenTextInstance(info.get(i), basePosX + 16, basePosY + 32 + i * 11));
+          textLines.add(new ScreenTextInstance(info.get(i), (basePosX + 16), ((basePosY + i + 32) * 11)));
```
Comment (TelepathicGrunt) [inline src/main/java/elucent/rootsclassic/client/screen/TabletPageScreen.java:145]: This changes the order of operations away from what it was before.

File: src/main/java/elucent/rootsclassic/client/screen/TabletPageScreen.java [hunk 2]
```diff
@@ -137,42 +135,43 @@ public void render(GuiGraphics guiGraphics, int mouseX, int mouseY, float partia
...
       case TYPE_SMELTING -> {
-        //        RenderSystem.setShaderTexture(0, Const.tabletSmelting);
+        //	RenderSystem.setShaderTexture(0, Const.tabletSmelting)
         guiGraphics.blit(Const.TABLETSMELTING, basePosX, basePosY, 0, 0, 192, 256);
         slots.add(new ScreenSlotInstance(page.smeltingRecipe.get(0), basePosX + 56, basePosY + 40));
         slots.add(new ScreenSlotInstance(page.smeltingRecipe.get(1), basePosX + 144, basePosY + 56));
         info = page.makeLines(makeInfo());
         for (int i = 0; i < info.size(); i++) {
-          textLines.add(new ScreenTextInstance(info.get(i), basePosX + 16, basePosY + 104 + i * 11, RenderUtil.intColor(255, 255, 255)));
+          textLines.add(new ScreenTextInstance(info.get(i), (basePosX + 16), ((basePosY + 104 + i) * 11), RenderUtil.intColor(255, 255, 255)));
```
Comment (TelepathicGrunt) [inline src/main/java/elucent/rootsclassic/client/screen/TabletPageScreen.java:157]: Same here. Order of operation changed.

File: src/main/java/elucent/rootsclassic/client/screen/TabletPageScreen.java [hunk 3]
```diff
@@ -137,42 +135,43 @@ public void render(GuiGraphics guiGraphics, int mouseX, int mouseY, float partia
...
       }
       case TYPE_DISPLAY -> {
-        //        RenderSystem.setShaderTexture(0, Const.tabletDisplay);
+        //	RenderSystem.setShaderTexture(0, Const.tabletDisplay)
         guiGraphics.blit(Const.TABLETDISPLAY, basePosX, basePosY, 0, 0, 192, 256);
         slots.add(new ScreenSlotInstance(page.displayItem, basePosX + 88, basePosY + 48));
         info = page.makeLines(makeInfo());
         for (int i = 0; i < info.size(); i++) {
-          textLines.add(new ScreenTextInstance(info.get(i), basePosX + 16, basePosY + 80 + i * 11, RenderUtil.intColor(255, 255, 255)));
+          textLines.add(new ScreenTextInstance(info.get(i), (basePosX + 16), ((basePosY + 80 + i) * 11), RenderUtil.intColor(255, 255, 255)));
```
Comment (TelepathicGrunt) [inline src/main/java/elucent/rootsclassic/client/screen/TabletPageScreen.java:168]: Order of operation changed.

File: src/main/java/elucent/rootsclassic/client/screen/TabletPageScreen.java [hunk 4]
```diff
@@ -202,20 +201,20 @@ public void render(GuiGraphics guiGraphics, int mouseX, int mouseY, float partia
...
-        //        RenderSystem.setShaderTexture(0, Const.tabletMortar);
+        //	RenderSystem.setShaderTexture(0, Const.tabletMortar)
         guiGraphics.blit(Const.TABLETMORTAR, basePosX, basePosY, 0, 0, 192, 256);
         title = makeTitle();
         if (page.mortarRecipe != null) {
           for (int i = 0; i < page.mortarRecipe.getIngredients().size(); i++) {
             Ingredient ingredient = page.mortarRecipe.getIngredients().get(i);
             if (ingredient.isEmpty()) {
-              slots.add(new ScreenSlotInstance(new ItemStack(Items.BARRIER), basePosX + 24 + i * 16, basePosY + 56));
+              slots.add(new ScreenSlotInstance(new ItemStack(Items.BARRIER), ((basePosX + 24 + i) * 16), basePosY + 56));
```
Comment (TelepathicGrunt) [inline src/main/java/elucent/rootsclassic/client/screen/TabletPageScreen.java:214]: Order of operation changed.

File: src/main/java/elucent/rootsclassic/client/screen/TabletPageScreen.java [hunk 5]
```diff
@@ -202,20 +201,20 @@ public void render(GuiGraphics guiGraphics, int mouseX, int mouseY, float partia
...
             }
             else {
-              slots.add(new ScreenSlotInstance(getStackFromIngredient(ingredient), basePosX + 24 + i * 16, basePosY + 56));
+              slots.add(new ScreenSlotInstance(getStackFromIngredient(ingredient), ((basePosX + 24 + i) * 16), basePosY + 56));
```
Comment (TelepathicGrunt) [inline src/main/java/elucent/rootsclassic/client/screen/TabletPageScreen.java:217]: Order of operation changed.

File: src/main/java/elucent/rootsclassic/client/screen/TabletPageScreen.java [hunk 6]
```diff
@@ -225,27 +224,25 @@ public void render(GuiGraphics guiGraphics, int mouseX, int mouseY, float partia
           }
           info = page.makeLines(makeInfo());
           for (int i = 0; i < info.size(); i++) {
-            textLines.add(new ScreenTextInstance(info.get(i), basePosX + 16, basePosY + 96 + i * 11, RenderUtil.intColor(255, 255, 255)));
+            textLines.add(new ScreenTextInstance(info.get(i), (basePosX + 16), ((basePosY + 96 + i) * 11), RenderUtil.intColor(255, 255, 255)));
```
Comment (TelepathicGrunt) [inline src/main/java/elucent/rootsclassic/client/screen/TabletPageScreen.java:227]: Order of operation changed.

**Comment by ErythroCraft** [comment]
Thank you @TelepathicGrunt.
This will help creating ChatGPT for Minecraft, to creating mods simpler.

**Pull request closed without merging**